packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -544,11 +557,13 @@ class PathNode implements UpPath {
     }
 
     /**
-     * Removes this from parent, and sets this to disposed.
+     * Removes this from parent if alive, and sets this to disposed.
```

Comment:
Since this function is a bit special, it could use more docs.

```suggestion
     * Removes this from parent if alive, and sets this to disposed.
     * Must only be called when this node is no longer needed (has no references and no children).
     *
     * Allowed when dangling (but not when disposed).
```